### ISSUE TITLE
EDIT: Request - List what specific data doesn't get blocked in the readme.

### ISSUE THREAD

**Issue by xCapRx** [PeterCxy/Shelter#74]
I assumed the work profile wouldn't block everything but apparently it misses a lot. According to this [Google support article](https://support.google.com/work/android/answer/6191949?hl=en), the following details are left exposed: 

Model, serial number, and ID
Phone Number
Carrier
Operating system
Build Number
Kernel Version
Baseband Version
MAC Address
Language

I'd like to recommend this be added to the caveats section of the readme.  Thanks.

**Comment by kakolisgay** [comment]
> Shelter **is not a full sandbox implementation**. It cannot protect you from:

> Any other bugs or limitations imposed by the Android system. (i.e. **If Android chooses to expose some information into the work profile, there is nothing I could do about it**)

This seems to already be in the readme

**Comment by xCapRx** [comment]
Sorry let me rephrase and request that these specific details be added to the readme since they're not in there.

**Comment by PeterCxy** [comment]
I have simply added the article to the README in case Google might update it at some point.
